source/stylesheets/layout/_header.scss

```diff
@@ -0,0 +1,162 @@
+@import "mediaqueries/media-header";
+@import "components/_button";
+
+// import fontello icons
+@import "icons/fontello";
+@import "icons/fontello-ie7";
+@import "icons/fontello-ie7-codes";
+@import "icons/fontello-codes";
+@import "icons/fontello-embedded";
+
+//services navigation styling
+.services-nav {
+  display: flex;
+  justify-content: space-between;
+  margin: $base-margin 0;
+  p {
+    color: $grey;
+    font-weight: 400;
+    margin: $base-margin;
+  }
+  .phone-number {
+    color: $turquoise;
+  }
+  .register-nav {
+    display: inline-flex;
+    height: 2em;
+    a {
+      font-size: $small-font;
+      font-weight: 400;
+      text-transform: uppercase;
+      color: $grey;
+      margin-top: $base-margin;
+      &:hover, &:focus {
+        text-decoration: none;
+      }
+      &:after {
+        content: "/";
+        font-weight: 900;
+        margin: 0 $base-margin;
+      }
+    }
+    .register {
```

Comment:
Improper naming ```register``` is more generic than ```register-nav``` so it shouldn't be nested. Change name ```register``` to be more specific